### ISSUE TITLE
AI-related work

### ISSUE THREAD

**Issue by xfq** [w3c/i18n-issues#11]
I've been observing the rapid advancements in Artificial Intelligence, like machine translation, speech recognition, and natural language processing. These technologies are rapidly becoming integral components of the web and how users interact with the web. To ensure the Web is truly usable around the world, regardless of language, writing system, or culture, we should actively consider how AI intersects with our work at the W3C Internationalization Activity.

AI models are increasingly deployed over the web, some are accessed via web APIs, and their outputs are integrated into web content. As AI permeates the web ecosystem, ensuring its compatibility with internationalization principles becomes really important to the web's global usability.

Our mission is to enable *everyone* to use web technologies. If AI, as it becomes more prevalent, introduces or exacerbates biases against low-resource languages, non-Latin scripts, or specific cultural contexts, it undermines the very foundation of the *worldwide Web* we champion.

These are some of the areas I initially thought of for your reference, and feedback is welcome:

1. Similar to [specdev](https://www.w3.org/TR/international-specs/), we can create a document for those building AI models and curating training data. It should contain guidance on actively balancing training data to include low-resource languages, diverse regional dialects, and non-Latin scripts.

2. We can propose metrics and test sets in various languages and scripts that specifically challenge AI on culturally specific idioms etc.

-----

Some interesting related work:

- [Multilingual Evaluations in LLMs — a comparison](https://scribe.rip/@vbsowmya/multilingual-evaluations-in-llms-a-comparison-1d58b0fd9848)
- [Language-Specific Neurons: The Key to Multilingual Capabilities in Large Language Models](https://arxiv.org/abs/2402.16438)

**Comment by xfq** [comment]
Another thing to consider is how to ensure that AI model developers, data scientists, ML engineers etc. follow these guidelines if we develop corresponding guidelines.

**Comment by dontcallmedom** [comment]
https://blog.mozilla.ai/towards-truly-multilingual-ai-breaking-english-dominance/ has a number of useful references re (1)

**Comment by dontcallmedom** [comment]
[example of a llm built for Mongolian language and culture](https://restofworld.org/2025/mongolia-egune-ai-llm/)

**Comment by r12a** [comment]
@dontcallmedom these are useful links, thanks!

Here are a couple more:

https://multilingual.com/multilingual-ai-assistant-refugees-aws/
https://multilingual.com/apple-ai-multilingual-access/

**Comment by xfq** [comment]
Related presentation: https://locworld.com/sessions/the-role-of-locally-relevant-content-in-evaluating-multilingual-llms/

**Comment by dontcallmedom** [comment]
https://www.technologyreview.com/2025/09/25/1124005/ai-wikipedia-vulnerable-languages-doom-spiral/ about the vicious cycle of poor machine translation of languages with limited corpus feeding poor LLM training via wikipedia

**Comment by r12a** [comment]
https://bezoku.ai/   an organisation that iiuc aims to help smaller languages through development of small language models

see also https://bsky.app/profile/bezoku.bsky.social

**Comment by xfq** [comment]
https://www.masakhane.io/ might also be related.

**Comment by xfq** [comment]
> https://bezoku.ai/ an organisation that iiuc aims to help smaller languages through development of small language models
> 
> see also https://bsky.app/profile/bezoku.bsky.social

Something interesting about it:

1. They are using bi-LSTMs, which are powerful architecture when training with sparse data. Transformers (generally) overfit in sparse data situations.
2. Their data pipeline is designed around the [CoNLL-U](https://universaldependencies.org/format.html) format. They handle orthographic features such as direction and structure as well as morphosyntactic features resulting from diacritical markings.

**Comment by xfq** [comment]
In [ATLAS: Adaptive Transfer Scaling Laws for Multilingual Pretraining, Finetuning, and Decoding the Curse of Multilinguality](https://arxiv.org/pdf/2510.22037) to be presented at [ICLR 2026](https://iclr.cc/), Google presents the largest public multilingual pre-training study to date, spanning 774 training runs across 10M–8B parameter models.

**Comment by dontcallmedom** [comment]
Another interesting summary of an event around AI and minority languages https://thoughtbot.com/blog/ai-and-minority-languages

**Comment by xfq** [comment]
> We can propose metrics and test sets in various languages and scripts that specifically challenge AI on culturally specific idioms etc.

If we want to establish evaluation benchmarks for multilingual models, here are some of my thoughts:

First, each language should have its own tasks, covering local expressions, knowledge, cultural common sense, etiquette systems, dialects, writing habits, and social sensitivities.

Second, Web internationalization compliance must be included in the evaluation. For example, when the model generates web pages, forms, JSON, captions, emails, and UI copy, does it correctly use `lang`, `dir`, numeric date formats, plural rules, name order, address formats, etc.? Our existing i18n checklists for spec developers, best practices regarding string languages ​​and directional metadata, etc., can all be transformed into AI evaluation items.

Third, we cannot only look at the average score. A model that performs well in 10 high-resource languages ​​but fails in 60 low-resource languages probably ​​should not be rated as a good multilingual model.

I don't yet have a concrete idea about how to design the specific questions. Designing only multiple-choice questions has the advantage of automatic scoring, but it might not simulate real-world scenarios well enough. However, we can't rely solely on human judges' intuition for scoring.

A more reasonable approach might be to use open-ended questions, with judging incorporating some automatically and semi-automatically verifiable hard metrics, and some human judgment.

It would be even better if we could involve some AI evaluation experts in this work.

Comments are welcome.
